mathjax3-ts/input/tex/color/ColorConfiguration.ts

```diff
@@ -0,0 +1,201 @@
+/*************************************************************
+ *
+ *  Copyright (c) 2018 Omar Al-Ithawi and The MathJax Consortium
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+
+/**
+ * @fileoverview Configuration file for the color package.
+ *
+ * @author v.sorge@mathjax.org (Volker Sorge)
+ */
+
+import {Configuration} from '../Configuration.js';
+import TexParser from '../TexParser.js';
+import {AllTags, NoTags} from '../Tags';
```

Comment:
Do you need that import? I don't think tags are used in the color extension.

Reply:
Removed it. Thanks!